README.md

```diff
@@ -1,8 +1,47 @@
 # elr143 News Server
 
-Please add the following files to connect to the databases:
+You can find the hosted server here: https://elr143-news-server.onrender.com
 
-.env.development -> this should contain PGDATABASE=nc_news
+This is an API which accesses application data programmatically. It mimics the backend functionality of a site such as Reddit and will provide information to the front end architecture.
+
+## Please add the following files to connect to the databases:
 
+.env.development -> this should contain PGDATABASE=nc_news
 .env.test -> this should contain PGDATABASE=nc_news_test
 
+### Setup Instructions:
+
+1. In your terminal, navigate to your chosen directory, then copy and paste the following:
+`git clone https://github.com/ELR143/elr143-news-server.git`
+
+2. Run `npm install`
+
+3. Check your version of Node.js is v20.5.0 or higher --> `node -v`
+
+4. Check the package.json file for the following **dependencies**. Ensure that your versions meet the minimums below:
+    - "dotenv": ^16.0.0
+    - "express": ^4.18.2
+    - "pg": ^8.7.3
+    - "pg-format": ^1.0.4
+
+5. Check the package.json file for the following **devDependencies**. Ensure that your versions meet the minimums below:
+    - "husky": ^8.0.2
+    - "jest": ^27.5.1
+    - "jest-extended": ^2.0.0
+    - "jest-sorted": ^1.0.14
+    - "supertest": ^6.3.3
```

Comment:
This isn't really a necessary step. When they clone your repo, this should all already be there as it is on main, so users won't have to manually check these dependencies